### ISSUE TITLE
nicehash keep freezing at starting excavator server

### ISSUE THREAD

**Issue by goldenboy36** [nicehash/NiceHashMiner-Archived#1369]
nicehash keep freezing at starting excavator server. no mather how many times i tried it just wont past that point. also when i log in it asked me to reset my password with email they send me but the thing is there is no email at all not even in the spam section

**Comment by ghost** [comment]
Yeah I'm noticing the same thing as well. It just sits there spinning trying to start excavator. Been doing that for about an hour now.

**Comment by VoltaicShock** [comment]
I am having the same issue.  It just sits on Starting Excavator Server...

**Comment by tinkyisretarded** [comment]
same problem for me. it does not start the excavator server for 45 minutes
[UPDATE1] also GPU benchmark is frozen and CPU cannot coonect to pool while benchmarking
[UPDATE2] on another computer it is mining with 0.000 H/s
![niceshit](https://user-images.githubusercontent.com/34728495/34238782-3bf49acc-e60c-11e7-88e7-5f04ca1c1b15.PNG)
[UPDATE3] also CPU mining is fucked on second computer
![niceshite](https://user-images.githubusercontent.com/34728495/34239164-373b2ee0-e60e-11e7-83b4-4f607d2c6462.PNG)

**Comment by RamiroSommelier** [comment]
Same 8=(

**Comment by ghost** [comment]
same thing to me

**Comment by Rockcena** [comment]
I also have same issue

**Comment by amit-feldman** [comment]
Same here

**Comment by ghost** [comment]
i have tried Legacy and without any problem...

**Comment by Nekroskop** [comment]
Golden if your MH its stuck on excavator server it may be you AV software if you have real-time scanning you should exclude MH so your AV will not block it. ON my PC i did have to exclude this 3 files to make it work:

excavator.exe
C:\Users\Admin\AppData\Roaming\nhm2\bin\excavator_server

XMR-STAK-CPU.EXE
C:\Users\Admin\AppData\Roaming\nhm2\bin\xmr-stak-cpu

NiceHash Miner 2.exe
C:\Users\Admin\AppData\Local\Programs\NiceHash Miner 2

I can start application since I did this run and finish all benchmarks  but my GPU mining still get disconnected and I have no idea how to fix this:

[09:41:02][0x00000438][info] core | Initialized!
[09:41:07][0x00001484][info] core | Found compatible devices
[09:43:45][0x00001484][info] net | Connecting to 159.8.13.248:3347 (lyra2rev2.eu.nicehash.com)
[09:43:45][0x00001484][info] net | Connected!
[09:43:45][0x000036d8][info] core | Algorithm 'lyra2rev2' total speed: 0.000000 H/s
[09:43:45][0x00001ca4][info] wrkr0-0 | Algorithm: CUDA-lyra2rev2 parameters: T=2810880[09:43:50][0x000036d8][info] core | Algorithm 'lyra2rev2' total speed: 0.000000 H/s
[09:43:55][0x000036d8][info] core | Algorithm 'lyra2rev2' total speed: 0.000000 H/s
[09:44:00][0x000036d8][info] core | Algorithm 'lyra2rev2' total speed: 0.000000 H/s
[09:44:04][0x000036d8][info] net | Connection lost! Reconnecting in 7 seconds
[09:44:05][0x000036d8][info] core | Algorithm 'lyra2rev2' total speed: 0.000000 H/s
[09:44:10][0x000036d8][info] core | Algorithm 'lyra2rev2' total speed: 0.000000 H/s
[09:44:11][0x00001484][info] net | Connecting to 159.8.13.248:3347 (lyra2rev2.eu.nicehash.com)
[09:44:11][0x00001484][info] net | Connected!
[09:44:15][0x00001484][info] core | Algorithm 'lyra2rev2' total speed: 0.000000 H/s
[09:44:20][0x00001484][info] core | Algorithm 'lyra2rev2' total speed: 0.000000 H/s
[09:44:25][0x00001484][info] core | Algorithm 'lyra2rev2' total speed: 0.000000 H/s
[09:44:30][0x00001484][info] core | Algorithm 'lyra2rev2' total speed: 0.000000 H/s
[09:44:30][0x00001484][info] net | Connection lost! Reconnecting in 7 seconds
[09:44:35][0x00001484][info] core | Algorithm 'lyra2rev2' total speed: 0.000000 H/s
[09:44:37][0x000036d8][info] net | Connecting to 159.8.13.248:3347 (lyra2rev2.eu.nicehash.com)
[09:44:37][0x000036d8][info] net | Connected!
[09:44:40][0x000036d8][info] core | Algorithm 'lyra2rev2' total speed: 0.000000 H/s
[09:44:45][0x000036d8][info] core | Algorithm 'lyra2rev2' total speed: 0.000000 H/s
[09:44:50][0x000036d8][info] core | Algorithm 'lyra2rev2' total speed: 0.000000 H/s
[09:44:55][0x000036d8][info] core | Algorithm 'lyra2rev2' total speed: 0.000000 H/s
[09:44:57][0x000036d8][info] net | Connection lost! Reconnecting in 6 seconds
[09:44:58][0x00001484][info] core | Mining Session END
